deploy/ansible/roles-sap/0.1-bom-validator/tasks/main.yaml

```diff
@@ -66,6 +66,9 @@
     bom_name:                          "{{ bom_base_name }}"
 # -------------------------------------+---------------------------------------8
 
+- name: Bom folder path
+  debug:
+    msg: "{{ sapbits_location_base_path }}/boms/{{ bom_base_name }}"
```

Comment:
```suggestion
    msg:                           "{{ sapbits_location_base_path }}/boms/{{ bom_base_name }}"
```

Reply:
> 

resolved in commit b50c03666a268e0527a05b30033c2ef206b31899